libs/partners/together/langchain_together/chat_models.py

```diff
@@ -0,0 +1,103 @@
+"""Wrapper around Together AI's Chat Completions API."""
+
+import os
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+)
+
+import openai
+from langchain_core.pydantic_v1 import Field, SecretStr, root_validator
+from langchain_core.utils import (
+    convert_to_secret_str,
+    get_from_dict_or_env,
+)
+from langchain_openai import ChatOpenAI
```

Comment:
Should inherit from `BaseChatOpenAI` instead

Reply:
Says langchain_openai doesn't have a module called that. Plus I saw this was the way upstage was doing it. Where should I import that from?

Reply:
`from langchain_openai.chat_models.base import BaseChatOpenAI`

Will ping the upstage folks to do the same. Basically if you rely on `ChatOpenAI` it will throw an error if you don't pass/have an environment variable for `OPENAI_API_KEY`. Using the base one removes that restriction, such that you can read `together_api_key` instead.

Reply:
Ahh thanks for the explanation! Done